frontend/src/App.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { describe, it, expect } from "vitest";
+import { render, screen } from "@testing-library/react";
+import App from "./App";
+
+describe("App component", () => {
+  it("renders the heading with correct text", () => {
+    render(<App />);
+
+    // Find the heading element
+    const headingElement = screen.getByRole("heading", { level: 1 });
+
+    // Check if heading has correct text
+    expect(headingElement.textContent).toBe("Working Title");
```

Comment:
since this text is bound to change, seems like the test should maybe check something that's not expected to change?

Reply:
Honestly I think the tests will be overhauled completely when we actually add anything useful. 
This is just to prove that "the concept of unit testing" works.